cmd/podman/commit.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
+	rt "github.com/projectatomic/libpod/cmd/podman/util"
```

Comment:
Why did you not use util?

Reply:
Some podman commands have util already imported so I was trying to differentiate.

Reply:
and util is imported later

Reply:
Can we add these functions to pkg/util then?  I am all confused by this util?  Or change the name of the file to some other name.  Is this package uitl?

Reply:
This is more of a podman-util (pkg/util/ is more libpod stuff). Though not that I think of it, we now have three packages named util in the project. We should do something about that.

Reply:
If these are shared only by cmd in podman directory then it should go in the utils.go file.

Reply:
They're in use by the Varlink package too, now.
Though we could still keep them in utils.go under Podman and just make them public? That sounds like a good compromise

Reply:
utils.go is in main ... i dont think that helps.

Reply:
Oh. I forgot that Go forces a main package in addition to a main function...

Reply:
Can we put util.go and utils.go into a subdir util and then put them in the same file package util?

Reply:
ok fixed up